runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -659,12 +613,197 @@ public synchronized CorfuRuntime connect() {
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setCacheDisabled(boolean disable) {
+        log.warn("setCacheDisabled: Deprecated, please set parameters instead");
+        parameters.setCacheDisabled(disable);
+        return this;
+    }
+
+    /**
+     * Whether or not to use the fast loader.
+     *
+     * @param enable True, if the fast loader should be used, false otherwise.
+     * @return A CorfuRuntime to support chaining.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setLoadSmrMapsAtConnect(boolean enable) {
+        log.warn("setLoadSmrMapsAtConnect: Deprecated, please set parameters instead");
+        parameters.setUseFastLoader(enable);
+        return this;
+    }
+
+    /**
+     * Whether or not hole filling is disabled
+     *
+     * @param disable True, if hole filling should be disabled
+     * @return A CorfuRuntime to support chaining.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setHoleFillingDisabled(boolean disable) {
+        log.warn("setHoleFillingDisabled: Deprecated, please set parameters instead");
+        parameters.setHoleFillingDisabled(disable);
+        return this;
+    }
+
+    /** Set the number of cache entries.
+     *
+     * @param numCacheEntries   The number of cache entries.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setNumCacheEntries(long numCacheEntries) {
+        log.warn("setNumCacheEntries: Deprecated, please set parameters instead");
+        parameters.setNumCacheEntries(numCacheEntries);
+        return this;
+    }
+
+    /** Set the cache expiration time.
+     *
+     * @param expiryTime   The time before cache expiration, in seconds.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setCacheExpiryTime(int expiryTime) {
+        log.warn("setCacheExpiryTime: Deprecated, please set parameters instead");
+        parameters.setCacheExpiryTime(expiryTime);
+        return this;
+    }
+
+    /** Set the bulk read size.
+     *
+     * @param size  The bulk read size.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setBulkReadSize(int size) {
+        log.warn("setBulkReadSize: Deprecated, please set parameters instead");
+        parameters.setBulkReadSize(size);
+        return this;
+    }
+
+
+    /** Set the write retry time.
+     *
+     * @param writeRetry   The number of times to retry writes.
+     * @deprecated  Deprecated, set using {@link CorfuRuntimeParameters} instead.
+     */
+    @Deprecated
+    public CorfuRuntime setWriteRetry(int writeRetry) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)